src/main/java/graphql/schema/CodeRegistryVisitor.java

```diff
@@ -12,11 +23,12 @@
  * This ensure that all fields have data fetchers and that unions and interfaces have type resolvers
```

Comment:
I moved this to a schem.impl package just to clean things up.  Its kinda of a private thing and not general purpose